plugins/resteasy-http/src/main/java/org/radargun/http/service/RESTEasyService.java

```diff
@@ -41,6 +44,14 @@
    @Property(doc = "Semicolon-separated list of server addresses.", converter = RESTAddressListConverter.class)
    protected List<InetSocketAddress> servers;
 
+   @Property(doc = "Ratio between the number of connections to individual servers. " +
+                   "Servers from the 'servers' list are indexed from 0. When the client " +
+                   "is first created it will choose a server to communicate with according " +
+                   "to this load balancing setting. The client will keep communicating with " +
+                   "this single server until redirected.", converter = Fuzzy.IntegerConverter.class)
+   //By default clients only connect to server 0. Example for two servers with equal load: "50%:0,50%:1"
+   protected Fuzzy<Integer> serversLoadBalance = Fuzzy.uniform(0);
```

Comment:
Use @Init annotation on a method, that will be called after properties are filled in, so then you can initialize this property (if it is still null)